tests/suites/0_stateless/14_clickhouse_http_handler/14_0005_clickhouse_federated.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+curl -s -u root: -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_PORT}" -d 'drop table if exists t1'
+
+curl -s -u root: -XPOST "http://localhost:${QUERY_CLICKHOUSE_HTTP_HANDLER_PORT}" -d 'create table t1(a int) engine=MergeTree'
```

Comment:
What about this issue

create table t (id int, id2 date) engine = mergetree partition by expr order by expr

Reply:
do we have to support it？

Reply:
I think this PR mainly for airbyte, partition by is not needed now.

Reply:
I guess we do not necessary need keeping SQL syntax compatible with the existed airbyte clickhouse connector, but fork a connector with a little modification to our SQL syntax is ok.

compares to keep the syntax SQL compatibility, building a connector in airbyte is not a hard work.

Reply:
other drivers may use clickhouse SQLs too.

https://github.com/xzkostyan/clickhouse-sqlalchemy/tree/master/clickhouse_sqlalchemy/engines

Reply:
So we can use a fork of airbyte clickhouse connector for now.  and postpone adding support compatible in query-server until necessary? @flaneur2020

Reply:
I think we can take it into serveral parts:

- to integrate BI: mostly user hand written SELECT statements, not much needs to keep syntax compatible
- to integrate ETL pipeline to ingest data: our forked connector is nesscary since cloud warehouses mostly favors STAGED upload (for free bandwidth fee). 
- to integrate SQL driver sdks: we can implement the on wire protocol to proccess the user SQL, but do not necessary needs a full featured ORM which requires CREATE TABLE which built-in the ORM library.